splunk_add_on_ucc_framework/ucc_ui_lib/src/main/webapp/components/table/CustomTable.jsx

```diff
@@ -70,15 +124,36 @@ function CustomTable({ isInput, serviceName, data, handleToggleActionClick }) {
         );
     };
 
-    const handleEditActionClick = () => {};
+    const handleEditActionClick = (row) => { 
+        const temEntity ={...entityModal}
```

Comment:
Can we also refactor this as the previous comment?

Reply:
done